package.json

```diff
@@ -2,7 +2,11 @@
 	"name": "plop",
 	"version": "2.6.0",
 	"description": "Micro-generator framework that makes it easy for an entire team to create files with a level of uniformity",
-	"main": "./src/plop",
+	"main": "./dist/plop",
```

Comment:
The source file is now built and should hopefully still work if anyone was using the entry point. @crutchcorn do you have a test for this?

Reply:
We have no tests for `plop` itself at this time. I don't think this would be a breaking change though

Reply:
Actually, I mention that, but we do document usage of `require('plop')` in some apps that wrap plop:

https://plopjs.com/documentation/#wrapping-plop

I can test this manually, I think

Reply:
![image](https://user-images.githubusercontent.com/9100169/118587257-24a59f00-b751-11eb-8e0e-17d563daad3a.png)

This PR seems like it may be minifying the `Plop` and `run` variable names

```javascript
(()=>{var e={id:341,ids:[341]};e.modules={81386:(e,i,o)=>{const r=process.argv.slice(2),{Plop:c,run:l}=o(46363),s=o(36101)(r);c.launch({cwd:s.cwd,configPath:s.plopfile,require:s.require,completion:s.completion},l)}};var i=require("../dist/plop.js");i.C(e),i(i.s=81386)})();
```

Even looking through the `bin` file, it seems like `c` is `Plop` while `l` is `run`.

We'll want to fix this before moving forward

Reply:
Fixed by using `commonjs2` as the `libraryTarget` for the entrypoint and not for the bundle with the runtime chunk